### ISSUE TITLE
APA would like a liaison statement

### ISSUE THREAD

**Issue by iherman** [w3c/vc-wg-charter#108]
[Internal issue](https://github.com/w3c/strategy/issues/279#issuecomment-1098098993) raised by @michael-n-cooper, in the name of the W3C WAI activity

> APA would like a liaison statement, worded something like "Coordinate on accessibility use cases for verifiable credentials, and work jointly on a successor publication to inaccessible CAPTCHAs."

("APA" stands for the W3C [Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/))

**Comment by Sakurann** [comment]
Could someone please help me understand what is “successor publication to inaccessible CAPTCHAs” and the synergy with VCs? I am sorry if I have missed some conversations, but I am lacking the context to approve the PR atm.

**Comment by iherman** [comment]
@michael-n-cooper the question is really for you. What I know is that the WAI team are working on accessible versions captcha-s (the current, visual captcha-s are horrible). I suspect some people in the APA group are considering VC-s a possible tool for these, and they would like the VC group to look over their shoulders... If this is indeed the case, it would be a great use case (and, to be honest, unbeknownst to me).

**Comment by michael-n-cooper** [comment]
@iherman @Sakurann this is picking up on a conversation at the Fukuoka TPAC:

https://www.w3.org/2019/09/20-webauthn-minutes.html

There aren't headings, search for "Janina: ... we have been looking at captcha" for the start of that topic.

APA has been exploring this in the Research Questions Task Force, and wants to work on a Note-track deliverable jointly with Verifiable Credentials on how "zero knowledge proof" can replace current CAPTCHAs.

@JaninaSajka @matatk

**Comment by iherman** [comment]
@michael-n-cooper this puts the original request in another perspective.

The WG does have experts on ZKP-s, that is clear. But the charter of the WG is _not_ on ZKP; VC-s are only "users" of ZKP-s. Actually, the charter is pretty explicit about the group not working on crypto primitives per se.

I am sure ZKP experts would be happy to talk to APA on its usage on CAPTCHAs. But if I misunderstood, and the idea is not to use VCs but a particular crypto primitive, then I do not think the liaison statement makes sense in the charter.
